vyper/functions/functions.py

```diff
@@ -636,6 +643,13 @@ def minmax(expr, args, kwargs, context, is_min):
         o = ['if', [comparator, '_l', '_r'], '_r', '_l']
         otyp = left.typ
         otyp.is_literal = False
+    elif not is_min and _can_compare_with_uint256(left) and _can_compare_with_uint256(right):
+        o = ['if', [comparator, '_l', '_r'], '_r', '_l']
+        if right.typ.typ == 'uint256':
+            otyp = right.typ
+        else:
+            otyp = left.typ
+        otyp.is_literal = False
     # elif left.typ.typ == 'int128' and right.typ.typ == 'decimal':
```

Comment:
This commented area can be taken out, for now I believe it's better we don't support implicit conversion.